modules/extensions/src/views/lite/components/debugger/style.scss

```diff
@@ -1,5 +1,44 @@
 @import '~@blueprintjs/core/lib/scss/variables';
 
+:root {
+  --font-family: Roboto;
+
+  // Albert variables
+  // We should start using these variables from now on and phase out the rest of this file
```

Comment:
why do we need to copy that here ? They already exist right ?

Reply:
For some reason styles were not applied, will need to ask Max why

Reply:
@MaxCloutier  ?

Reply:
I believe it's because it's in an iframe and the variables are not accessible from there